### ISSUE TITLE
Number of GPUs for Pretraining and Model

### ISSUE THREAD

**Issue by Joan947** [facebookresearch/sonata#11]
You recommend we use 32 GPUs for training. What type and model of GPU  did you use for the training. I currently have 8 nvidia l40s available. Will it be an issue for pretraining Sonata? Are there any tips for this case?

**Comment by Gofinge** [comment]
I use a100, yet I think I40 with 48 memory should also work. If OOM occur, you can half the batch size and learning rate.

**Comment by Joan947** [comment]
Thanks for the feeback, I used 8 h100s with half the batch size which helped

**Comment by Gofinge** [comment]
Don’t forget also half the learning rate.
